src/display/editor/drawers/freedraw.js

```diff
@@ -734,9 +411,53 @@ class FreeOutliner {
       this.#isLTR
     );
   }
+
+  _getOutlineStart(outline, pos) {
+    const bottom = this.#bottom;
+    outline.set([NaN, NaN, NaN, NaN, bottom[4], bottom[5]], pos);
+    return (pos += 6);
+  }
+
+  _getOutlineEnd(outline, pos) {
```

Comment:
Can we make this method actually private instead?
```suggestion
  #getOutlineEnd(outline, pos) {
```